### PR TITLE
Reword FileSystem documentation

### PR DESCRIPTION
**Pull request by pachun**
The documentation for `FileSystem.uploadAsync` looks like it may have
been copied from the `FileSystem.downloadAsync` documentation and a
small update that should have been made was overlooked. The word
"download" appears several times in the `uploadAsync` method's
description. This commit rewords those statements to say:

> ... the upload network request ...

rather than

> ... the download network request ...

### CODE REVIEW

File: docs/pages/versions/v38.0.0/sdk/filesystem.md
Comment (barthap) [inline docs/pages/versions/v38.0.0/sdk/filesystem.md:366]:
```suggestion
- **status (_number_)** -- The HTTP response status code for the upload network request.
```

File: docs/pages/versions/v38.0.0/sdk/filesystem.md
Comment (barthap) [inline docs/pages/versions/v38.0.0/sdk/filesystem.md:368]:
```suggestion
- **headers (_object_)** -- An object containing all the HTTP response header fields and their values for the upload network request. The keys and values of the object are the header names and values respectively.
```

**Review by barthap — COMMENTED** [review]
Thank you for contribution! 🎉 
I've noticed that both `downloadAsync` and `uploadAsync` return values description refers to _"request"_, but of course it should be _"response"_ 😉  I marked some places, but it's not all of them.

**Comment by pachun** [comment]
@barthap Thanks for your feedback.

I included your suggested changes in a separate commit. Happy to squash those before merging. I'm not sure which you prefer.

I also rebased the branch back onto the current version of master, hence the force push.

Let me know what you think and thanks for the help!

**Review by barthap — APPROVED** [review]
Hi, you don't have to squash commits, they're gonna be squashed automatically during merge into one commit named the same as the PR title.

**Comment by pachun** [comment]
Ok, good to know!

I updated the PR title to be less specific to the first commit, and more describe the whole of the changes.

**Comment by pachun** [comment]
@barthap Just checking in to make sure nobody is waiting for me to do something else in order to merge this. If not, great! If so, happy to do it! Just let me know what needs to be done!

**Comment by barthap** [comment]
There's nothing else to do here now. We need to just wait for someone else approving your PR and merging it.

Thanks again for your contribution, I'll be glad to see your further PRs!  🎉

**Comment by barthap** [comment]
Hello again @pachun

I'm sorry nobody else reacted to your PR for so long - everybody is consumed with SDK 39 release process.

Now there is another page where your change needs to be done - in `docs/pages/versins/39.0.0`. The simplest way is probably to rebase your branch on current master.

**Comment by pachun** [comment]
@barthap I rebase on the current master and made the requested changes. Double check me and make sure I didn't goof anything, please!

I have been discouraged that this hasn't been merged yet, but given the understanding is a preoccupation with deploying SDK 39, I want to say "carry on".

SDK updates are little Christmases throughout the year in my world.

Thanks for the feedback and direction!

**Comment by barthap** [comment]
Looks good to me 🎉  I'll take care of this to be merged soon.

Tip for you (if you haven't known this already) - you can cd into `docs/` directory, run `yarn && yarn dev` to start documentation server locally. Then you can view your changes live by opening `http://localhost:3000` in your browser. More info [here](https://github.com/expo/expo/blob/master/CONTRIBUTING.md#-updating-documentation). Hope it helps in future contributions!

**Comment by pachun** [comment]
@barthap Thank's so much! This is pretty exciting! My name's on Expo!

Thanks again!

**Pull request merged**
